### DISCUSSION TITLE
Let the new Emote Modifiers work with 7TV, FFZ, and Twitch emotes

### DISCUSSION THREAD

**Discussion by Wulfestwulf** [night/betterttv#5909] [Ideas]
Currently, the new Emote Modifiers only work with BTTV emotes and not any of the other emote suppliers as listed above.
Edit: It might also be great if it could work with FFZ and 7TV installed, because currently you have to deactivate the extensions in order to see the modified emotes, and currently any chat view on stream won't display the modified emotes.

**Comment by night** [comment]
The emotes already do work with any emote BetterTTV is in control of rendering, including FFZ and 7TV emotes. However, because Twitch is in charge of rendering its own emotes we do not support them today. We may be able to figure out a way to make that work with time.

**Reply by Wulfestwulf** [reply]
The FFZ and 7TV Emotes are not getting the emote modifiers applied to them, even with just BTTV installed. Also, if 7TV is enabled, the emote modifiers are not being applied. This is likely on their side, but it is worth noting. And by the sound of things, it sounds like FFZ and 7TV emotes not getting the modifiers applied is a bug. I'll make an issue about it.

**Reply by night** [reply]
We've already verified any emotes rendered by BetterTTV apply modifiers properly. If you are not seeing modifiers for other emotes, you may need to disable those respective extensions and try again.

**Reply by Wulfestwulf** [reply]
Alright, I realize what was happening. But, I have now learned that 7TV completely overides/removes the emote modifiers from working, even for BTTV emotes.
